aip/general/0193.md

```diff
@@ -62,11 +62,109 @@ a [`PreconditionFailure`][PreconditionFailure].
 **Note:** `ErrorInfo` represents a special case. There **must** be exactly one
 `ErrorInfo`. It is required.
 
-### Localization
-
-Error messages **must** be in English. If a localized error message is also
-required, the service **should** use [`google.rpc.LocalizedMessage`][details]
-in the `details` field.
+### Error messages
+
+For each error, the service **must** populate the `message` field on [`google.rpc.Status`][Status]. This error message,
+
+- is developer-facing, human-readable, and in the application's
+  native language
+- both explains the error and offer an actionable resolution to it
+  ([citation](https://cloud.google.com/apis/design/errors#error_model))
+
+**Note:** Sometimes, services elect to publish debug error message in
+English rather than the application's native language so that messages
+are more easily universally searchable in common online knowledge bases.
+
+For new errors, the payload **must** include `ErrorInfo` and any
```

Comment:
New errors as a new instance of an existing error type? Or for new error _types_?

maybe s/types/modes/ idk

Reply:
It's kind of left intentionally vague, but it is when you would introduce a new value for "reason" in `ErrorInfo`. Not, when you would add an `ErrorInfo` to an error with an existing error message and code pair, but an entirely new error scenario. Any help with the wording given that would be much appreciated.

Reply:
> When introducing a new error, a failure scenario that did not previously occur for the service, the payload **must** include `ErrorInfo` and any variables found in dynamic segments of the error message **must** be represented in `metadata`.

wdyt?

Reply:
I pushed this change. Feel free to "unresolve".